package.json

```diff
@@ -22,14 +22,13 @@
     "compile": "npm run compile --workspaces --if-present",
     "test": "npm run test --workspaces --if-present",
     "coverage": "npm run coverage --workspaces --if-present",
+    "clean":"npm run clean --workspaces --if-present",
     "docs": "typedoc"
   },
   "workspaces": [
-    "packages/sdk-core",
-    "packages/aptos",
-    "packages/evm",
-    "packages/solana",
-    "packages/sei",
-    "packages/sui"
+    "core/base",
+    "core/definitions",
+    "core/connect",
+    "platforms/evm"
```

Comment:
Missing the other platforms?

Reply:
others wont build yet with new backend stuff